lib/tasks/argo.rake

```diff
@@ -19,6 +19,7 @@ task :default => [:ci]
 
 task :ci do
   ENV['RAILS_ENV'] = 'test'
+  WebMock.allow_net_connect!
```

Comment:
Unfortunately yep its required here too when the bundle requires the gems, it will disable all connections. Still trying to figure all of this out.